shared/style/edit_mode.css

```diff
@@ -122,7 +115,3 @@
   background-color: #e51e1e;
   color: #fff;
 }
-
-[role="dialog"][data-type="edit"] > menu button:first-child {
```

Comment:
mmm looks like this rule is at least required by Contacts [1] or I missing something?

[1] https://github.com/azasypkin/gaia/blob/a6881205deae450757a8d1e1ed65e5e5be0ec633/apps/communications/contacts/index.html#L234